### PR TITLE
Spacing/Syntax fixes

### PR DESCRIPTION
**Pull request by chris18890**
Just a load of small spacing/syntax fixes that I emant to push ebfore, will elave open for a week for comments ebfore merging @ramgarden / @elplatt

### CODE REVIEW

**Comment by chris18890** [comment]
@ramgarden not fine enough for my liking, went back over it again :P

File: crm/modules/payment/payment.inc.php [hunk 1]
```diff
@@ -219,7 +219,7 @@ function payment_format_currency ($value, $symbol = true) {
                 $pence = sprintf('%02d', $count);
             }
             if ($symbol) {
-                $result .= '�';
+                $result .= '£';
```
Comment (ramgarden) [inline crm/modules/payment/payment.inc.php]: What changed here?  It's not obvious what changed.

Comment (chris18890) [reply crm/modules/payment/payment.inc.php]: the old symbolw asin't encoded porperly, so wouldn't render

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Seltzer CRM 0.5.7 - An open source CRM for hackerspaces  
+Seltzer CRM 0.5.8 - An open source CRM for hackerspaces
```
[the remark this answers is not in the document]
Comment (chris18890) [reply README.md]: have a look in /include/crm.inc.php, or if you're running a recent build it'll be on the bottom of the web page :)

File: crm/modules/member/form.inc.php [hunk 3]
```diff
@@ -578,27 +544,22 @@ function member_membership_delete_form ($sid) {
...
-*/
+ */
 function member_filter_form () {
-    
     // Available filters    
     $filters = array(
         'all' => 'All'
-        , 'voting' => 'Voting'
         , 'active' => 'Active'
+        , 'voting' => 'Voting'
```
[the remark this answers is not in the document]
Comment (chris18890) [reply crm/modules/member/form.inc.php]: It does, also makes it consisten with lines 339-349 (command_member_filter) in /modules/member/command.inc.php

File: crm/modules/amazon_payment/ipn.php [hunk 4]
```diff
@@ -41,7 +41,7 @@
 $http_params = implode('&', $clauses);
 // Generate the request parameters
 $q = 'Action=VerifySignature'
-    . '&UrlEndPoint=' . rawurlencode('http://' . $config_host . base_path() . 'modules/amazon_payment/ipn.php')
+    . '&UrlEndPoint=' . rawurlencode('https://' . $config_host . base_path() . 'modules/amazon_payment/ipn.php')
```
[the remark this answers is not in the document]
Comment (chris18890) [reply crm/modules/amazon_payment/ipn.php]: _possibly_, if they've set things up correctly the http link would just redirect to https; thena gain, it's supposed to be using https as per https://developer.amazon.com/docs/amazon-pay-onetime/ipn-overview.html

**Pull request merged**